jupyter/minimal/ubi8-python-3.8/Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+LABEL name="odh-notebook-jupyter-minimal-ubi8-python-3.8" \
+    summary="Minimal jupyter notebook image for ODH notebooks" \
+    description="Minimal jupyter notebook image with base Python 3.8 builder image based on UBI8 for ODH notebooks" \
+    io.k8s.display-name="Minimal jupyter notebook image for ODH notebooks" \
+    io.k8s.description="Minimal jupyter notebook image with base Python 3.8 builder image based on UBI8 for ODH notebooks" \
+    authoritative-source-url="https://github.com/guimou/custom-notebooks" \
+    io.openshift.s2i.build.commit.ref="main" \
+    io.openshift.s2i.build.source-location="https://github.com/opendatahub-io/notebooks/tree/main/base/jupyter/minimal/ubi8-python-3.8" \
+    io.openshift.s2i.build.image="quay.io/opendatahub/notebooks:jupyter-minimal-ubi8-python-3.8"
+
+USER 0
+
+WORKDIR /opt/app-root/src
+
+COPY requirements.txt .
+
+COPY start-notebook.sh .
+
+RUN python -m pip install -r requirements.txt
+
+RUN chmod 777 start-notebook.sh
+
+ENTRYPOINT ["/opt/app-root/src/start-notebook.sh"]
+
+CMD ["--port=8888", "--ip=0.0.0.0"]
+
+USER 1001
+
+WORKDIR /opt/app-root/home
```

Comment:
This can be simplified by doing:

```Dockerfile
COPY requirements.txt start-notebook.sh .

RUN python -m pip install -r requirements.txt && rm -f requirements.txt

ENTRYPOINT ["/opt/app-root/src/start-notebook.sh"]
```

Make sure the start-notebook script has exec permissions in your computer (permissions are copied along with the file inside the container image):

```
chmod +x jupyter/minimal/ubi8-python-3.8/start-notebook.sh 
```